src/FSharpPlus/Extensions/Dictionary.fs

```diff
@@ -149,3 +149,16 @@ module Dictionary =
     let intersect (source1: Dictionary<'Key, 'T>) (source2: Dictionary<'Key, 'T>) =
         intersectWith (fun a _ -> a) source1 source2
     #endif
+    
+    /// <summary>Applies given function to each value of the given dictionary.</summary>
+    /// <param name="f">The mapping function.</param>
+    /// <param name="x">The input dictionary.</param>
+    ///
+    /// <returns>Dictionary with values x for each dictionary value where the function returns Some(x).</returns>
+    let chooseIndex f (x: IDictionary<'Key, 'T>) =
```

Comment:
Which names do we use for extensions?
I think we can either use `choosei` and eventually for Maps and Dictionaries simply `choose`.
`ChooseIndex` can be set to compiled name with the attribute for those named `choosei`.
Any thoughts on this?

Reply:
renamed all extensions to `choosei`